uaaextras/templates/email/body-expiring-password.html

```diff
@@ -11,7 +11,7 @@
             will expire in {{password.daysUntilExpiration}} days.
         </p>
         <p>
-            Please <a href="{{password.changeLink}}">change your password</a> soon, before it expires.
+            Please <a href="https://login.fr.cloud.gov/login">log in</a> with your current password</a> and then <a href="{{password.changeLink}}">change your password</a>.
```

Comment:
This login link should probably be configurable. Will update the code when I get a chance.